api-reference/beta/api/directory_delete.md

```diff
@@ -0,0 +1,54 @@
+# Delete deletedItem
+
+Permanently delete an item in [deleted items](../resources/directory.md).
```

Comment:
The names of the files here look weird.  I'd have expected this to be directoryObject.md or deletedItem.md